core/src/main/java/org/apache/calcite/util/format/FormatElementEnum.java

```diff
@@ -201,6 +201,14 @@ public enum FormatElementEnum implements FormatElement {
       // TODO: how to support timezones?
     }
   },
+  W("w", "The week number of the month (Sunday as the first day of the week) as a decimal "
```

Comment:
I think the javaFormat of `week of month` should be `W` instead of `w`.

Reply:
done